components/ManualPreview/ManualPreview.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import Link from 'next/link'
+import cn from 'classnames'
+import Image from 'next/image'
+
+import styles from './ManualPreview.module.css'
+
+function ManualPreview({ title, subtitle, pageUrl, color, status, publishedDate, pattern, image }) {
+    return (
+        <Link
+            href={{
+                pathname: '/[[...pageUrl]]',
+                query: { pageUrl: [pageUrl] },
+            }}
+            className={styles.manual}
+        >
+            <div
+                className={styles.manualInner}
+                style={{ color, backgroundImage: pattern ? `url(${pattern})` : null }}
```

Comment:
I would also put it in a constant, but I can’t think of a name yet 😅

Reply:
done